src/main/java/com/baidu/hugegraph/structure/schema/PropertyKey.java

```diff
@@ -34,12 +35,17 @@
     private DataType dataType;
     @JsonProperty("cardinality")
     private Cardinality cardinality;
+    @JsonProperty("aggregate_type")
+    private AggregateType aggregateType;
```

Comment:
keep compatible with v0.9:

1. read schema from v0.9
2. write schema to v0.9 (optional)